CCDB/src/runConditionsClient.cxx

```diff
@@ -1,80 +1,84 @@
 /********************************************************************************
  *    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
  *                                                                              *
- *              This software is distributed under the terms of the             * 
- *         GNU Lesser General Public Licence version 3 (LGPL) version 3,        *  
+ *              This software is distributed under the terms of the             *
+ *         GNU Lesser General Public Licence version 3 (LGPL) version 3,        *
  *                  copied verbatim in the file "LICENSE"                       *
  ********************************************************************************/
 /**
  * runConditionsClient.cxx
  *
  * @since 2013-04-23
- * @author D. Klein, A. Rybalchenko
+ * @author D. Klein, A. Rybalchenko, R. Grosso, C. Kouzinopoulos
  */
 
 #include <iostream>
 
+#include "CCDB/ConditionsMQClient.h"
 #include "FairMQLogger.h"
 #include "FairMQParser.h"
 #include "FairMQProgOptions.h"
 
-#include "CCDB/ConditionsMQClient.h"
+#ifdef NANOMSG
+#include "FairMQTransportFactoryNN.h"
+#else
+#include "FairMQTransportFactoryZMQ.h"
+#endif
 
```

Comment:
So SetTransport(transportFactory) is not needed at all? When i try to remove it i get the following error:

Transport not initialized. Did you call SetTransport()?

Also, i grepped through the source code and it seems that the same code is used by other devices as well so probably these have to be updated too